service/src/main/java/uk/gov/mca/beacons/api/beaconuse/domain/BeaconUse.java

```diff
@@ -300,6 +300,10 @@ public String getName() {
   public Map<String, String> getCommunicationTypes() {
     Map<String, String> communicationTypes = new HashMap<String, String>();
 
+    if (BooleanUtils.isTrue(vhfRadio)) {
+      communicationTypes.put("VHF", "");
+    }
+
```

Comment:
Are more communication types likely to emerge in future? This might become difficult to maintain

Reply:
This is the full list now, fair point, but maybe we can think about changing it in the future if more are added